web_external/js/views/body/MapPanel.js

```diff
@@ -11,6 +11,10 @@ minerva.views.MapPanel = minerva.View.extend({
     _specifyWmsDatasetLayer: function (dataset, layer) {
         var minervaMetadata = dataset.getMinervaMetadata();
         var baseUrl = minervaMetadata.base_url;
+        if (minervaMetadata.hasOwnProperty('credentials')) {
+            baseUrl = '/wms_proxy/' + encodeURIComponent(baseUrl) + '/' +
+                    minervaMetadata.credentials;
+        }
         var layerName = minervaMetadata.type_name;
         // TODO: inclued projection in params ??
```

Comment:
Can you remove this comment line also, the projection is included in the params.